src/index.html

```diff
@@ -4,9 +4,347 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="Company-logo"
+          class="photo-logo"
+          width="40"
+          height="40"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+
+    <main class="container">
+      <div class="product" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="product"
+          class="product__photo"
+        >
+        <h2 class="product__title"> APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
```

Comment:
do to use tags like br - text could be different
```suggestion
        <h2 class="product__title"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```